libbeat/docs/breaking.asciidoc

```diff
@@ -9,9 +9,152 @@ changes, but there are breaking changes between major versions (e.g. 5.x to
 
 See the following topics for a description of breaking changes:
 
+* <<breaking-changes-6.3>>
 * <<breaking-changes-6.0>>
 * {auditbeat}/auditbeat-breaking-changes.html[Breaking changes in Auditbeat 6.2]
 
+[[breaking-changes-6.3]]
+=== Breaking changes in 6.3
+
+This section discusses the main changes that you should be aware of if you
+upgrade the Beats to version 6.3. Please also review the relevant
+Breaking Changes sections of the <<release-notes,release notes>>.
+
+[[breaking-changes-mapping-conflict]]
+==== New `host` namespace may cause mapping conflicts for Logstash
+
+// REVIEWERS: I describe these as mapping conflicts throughout, but the acutal
+// error message says mapper parsing exception. I'm wondering if I should use
+// that instead?
+
+This breaking change applies only to users who send Beats events to Logstash.
+
+Starting with version 6.3, Beats provides an `add_host_metadata` processor for
+adding fields, such as `host.name` and `host.id`, to Beats events. These fields
+are grouped under a `host` prefix and conform to the
+https://github.com/elastic/ecs[Elastic Common Schema (ECS)]. The `host` object
```

Comment:
I hope it's OK to call this the `host` object because saying `host` field doesn't convey the idea of the type being different.